native-platform/build.gradle

```diff
@@ -25,13 +24,14 @@ javadoc {
 
 library {
     baseName = 'native-platform'
-    sources.configureEach(CppSourceSet) {
-        source.from('src/shared/cpp', 'src/main/cpp')
-        headers.from('src/shared/headers')
-    }
+    cppSources.setFrom('src/shared/cpp', 'src/main/cpp')
+    privateHeaders.setFrom('src/shared/headers')
     variants.configureEach {
         if (NcursesVersion.values().any { buildVariant.hasAxisOf(it) }) {
             baseName = 'native-platform-curses'
+            sources.configureEach(dev.nokee.language.cpp.CppSourceSet) {
```

Comment:
❓ It would be important to discuss how plugin authors are expected to create default imports. We experimented with various solutions but they range in complexity and efficiency. A core feature would be much better.